Makefile-az.mk

```diff
@@ -38,6 +38,18 @@ az-mkacr: az-mkrg ## Create test ACR
 		--sku Basic --admin-enabled -o none
 	az acr login  --name $(AZURE_ACR_NAME)
 
+az-acrimport: ## Imports an image to an acr registry
```

Comment:
maybe `az-acrimport-pause` since its hardcoded what image is being imported here?